### DISCUSSION TITLE
Its 5 months since an update?

### DISCUSSION THREAD

**Discussion by JroachAu** [brianpetro/obsidian-smart-connections#370] [General]
Hi Brian & all,
I'm new to using ChatGPT and other LLMs within the context of Obsidian.
Brian, I'm interested to know if you're still working on this plugin to improve it, or if you've moved onto better things.
Or @All, maybe the plugin works perfectly as is?
I notice that there aren't that many enhancement requests anyway.
Cheers, KR

**Comment by berot3** [comment]
Im currently testing it. Besides some hiccups it works quite well. 

The maker recently commented here in the discussions that he is busy with an other project. But will come back with updates.
